src/cryptography/x509/name.py

```diff
@@ -138,6 +168,23 @@ def rfc4514_string(self) -> str:
             _escape_dn_value(self.value),
         )
 
+    @classmethod
+    def from_rfc4514_string(cls, value: str) -> "NameAttribute":
+        """
+        Parse a RFC4514 formatted Distinguished Name string.
+        """
+        attr_type, attr_value = value.split("=", 1)
```

Comment:
What if the `=` is inside a quoted string like `CN="Joe = Public",...`

Reply:
Not a problem due to the `maxsplit` argument. However it will raise an unhandled ValueError if `value` happens to contain no `=` at all. I cought the error and raised an appropriate error.